app/controllers/accounts_controller.rb

```diff
@@ -99,6 +101,10 @@ def username_param
     params[:username]
   end
 
+  def rss_url
+    "#{TagManager.instance.url_for(@account)}#{Addressable::URI.parse("/tagged/#{params[:tag]}").normalize if params[:tag].present?}.rss"
```

Comment:
I am not super comfortable with how this RSS-related code is implemented, so perhaps it could be separated into a different PR, so we can get the ActivityPub stuff in quicker?

Reply:
OK. Divide into another PR.

Reply:
It moved to https://github.com/tootsuite/mastodon/pull/10502